doc/working-with-pm.md

```diff
@@ -8,8 +8,8 @@ an alternate package manager.
 
 <!-- prettier-ignore -->
 {! ../package.json extract: {start: 'scripts', stop: '},', replace: [
-    ['help:(\D*)".*".*echo (.*)"', '### npm run \1\n\2'], # one-liner like test
-    '"help:\D*:\d*".*echo (.*)"', # a numbered continuation line of help
-    ['"help:(\D*)":.*"', '### npm run \1\n'], # header of a block of help lines
+    ['help:(.*\D)".*".*echo (.*)"', '### npm run \1\n\2'], # 1-liner help
+    '"help:.*[-:]\d*".*echo \\?"?(.*?)\\?"?"', # a continuation line of help
+    ['"help:(.*\D)":.*"', '### npm run \1\n'], # header of block of help lines
     '.' # ignore the actual scripts that don't start with help
 ]} !}
```

Comment:
ditto

Reply:
Here is an example in which we have been using the inclusion syntax  for months (if not a couple years now) -- you can look at this file in the GitHub code browser for the "main" branch and see that there it does not actually show the commands you can run npm with. And that hasn't been deemed a problem before. But in the end, we need to decide where it should be possible to access fully readable documentation and proceed accordingly -- I won't respond to any of the other "dittos" until we make that decision. We could discuss at tomorrow's meeting.

Reply:
Glen is going to add an issue and not address this in this PR.  Resolving.